extensions/vscode/src/granite/panels/setupGranitePage.ts

```diff
@@ -83,17 +83,12 @@ export class SetupGranitePage {
           this.wizardState.stepStatuses[MODELS_STEP];
         let reopen = false;
         if (!isComplete) {
-          const RESUME_LABEL = "Resume Setup";
-          const choice = await window.showWarningMessage(
-            "Resume Granite.Code Setup?",
-            {
-              modal: true,
-              detail:
-                "Granite.Code needs to be setup before it can be used. Setup is not yet complete.",
-            },
-            RESUME_LABEL, // Cancel is always shown
+          const REOPEN_LABEL = "Open Setup";
+          const choice = await window.showInformationMessage(
+            "Granite.Code setup incomplete",
```

Comment:
Let's make this "Granite.Code setup is incomplete"

Reply:
done